python/nammu/controller/ModelController.py

```diff
@@ -0,0 +1,95 @@
+'''
+Created on 17 Apr 2015
+
+Displays ATF model view in a separate window.
+It'll possibly be shown instead of the text view in the AtfAreaView in the 
+future.
+
+@author: raquel-ucl
+'''
+
+from java.util import Vector
+
+from ..view.ModelView import ModelView
+from pyoracc.model.translation import Translation
+from pyoracc.model.ruling import Ruling
+from pyoracc.model.comment import Comment
+from pyoracc.model.line import Line
+
+class ModelController():
+    
+    def __init__(self, mainControler, parsedAtf):
+        """
+        1. Parse text area content
+        2. Change atfArea mode to model view from parent controller?
+        3. Process parsed data and serialize in separate JPanel
+        4. Think about whether the other user options should remain visible or
+        should this just be shown in a separate window?
+        """
+        
+        #Create view with a reference to its controller to handle events
+        self.view = ModelView(self)
+        
+        #Will also need delegating to parent presenter
+        self.controller = mainControler
+        
+        #Get ATF object parsed from text displated in text area
+        self.atf = parsedAtf
+        
+        #Go through parsed ATF object, serialize and pass elements to view
+        atfText = parsedAtf.text
+        
+        #Add a new tab in the view per object in the text
+        objectID = "&" + atfText.code + " = " + atfText.description
+        self.view.addObject(objectID)
+        #TODO: ATF protocols are not saved in the model yet, but need to be passed here:
+        #self.view.objectTab[objectID].addMetadata(objectID, atfText.project, \
+        #                                          atfText.language, \
+        #                                          atfText.protocols)
+        self.view.addMetadata(objectID, atfText.project, \
+                                                  atfText.language)
+
+        for item in atfText.children:
```

Comment:
Can't be done yet - this was needed because pyoracc was not working yet and I needed to have a first version for the GUI. I'm postponing this change until I fix pyoracc first.